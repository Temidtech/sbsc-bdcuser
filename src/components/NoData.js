import React, { PureComponent } from 'react';
import { View, Text } from 'react-native';
import { HistoricRatesStyles as styles } from '../styles';

const noDataText = 'No data available for this location';

export default class NoData extends PureComponent {
  render() {
    return (
      <View style={styles.noDataView}>
        <Text
          style={
            [
              styles.errorText,
              styles.noDataText
            ]
          }>
          {noDataText}
        </Text>
      </View>
    );
  }
}
