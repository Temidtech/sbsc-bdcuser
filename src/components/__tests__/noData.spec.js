import React from 'react';
import renderer from 'react-test-renderer';
import NoData from '../NoData';

describe('<NoData />', () => {

  it('renders the NoData component correctly', () => {
    const tree = renderer.create(
      <NoData />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
