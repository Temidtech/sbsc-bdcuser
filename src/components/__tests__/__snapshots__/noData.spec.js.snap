// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoData /> renders the NoData component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "height": 400,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "red",
          "fontFamily": "OpenSans-SemiBold",
        },
        Object {
          "color": "white",
          "fontSize": 20,
          "paddingTop": 160,
        },
      ]
    }
  >
    No data available for this location
  </Text>
</View>
`;
